analyzers/its/expected/Ember-MM/Ember Media Manager-{9B57D3AB-AF12-4012-B945-284C2448DC81}-S3900.json

```diff
@@ -0,0 +1,264 @@
+{
+"issues":  [
+{
+"id":  "S3900",
+"message":  "Refactor this method to add validation of parameter 'sFormula' before using it.",
+"location":  {
+"uri":  "sources\Ember-MM\Ember%20Media%20Manager\clsTheming.vb",
+"region":  {
+"startLine":  164,
+"startColumn":  20,
+"endLine":  164,
+"endColumn":  28
+}
+}
+},
+{
+"id":  "S3900",
+"message":  "Refactor this method to add validation of parameter 'sFormula' before using it.",
+"location":  {
+"uri":  "sources\Ember-MM\Ember%20Media%20Manager\clsTheming.vb",
+"region":  {
+"startLine":  169,
+"startColumn":  36,
+"endLine":  169,
+"endColumn":  44
+}
+}
+},
+{
+"id":  "S3900",
+"message":  "Refactor this method to add validation of parameter 'tTheme' before using it.",
+"location":  {
+"uri":  "sources\Ember-MM\Ember%20Media%20Manager\clsTheming.vb",
+"region":  {
+"startLine":  231,
+"startColumn":  74,
+"endLine":  231,
+"endColumn":  80
+}
+}
+},
+{
+"id":  "S3900",
+"message":  "Refactor this method to add validation of parameter 'tTheme' before using it.",
+"location":  {
+"uri":  "sources\Ember-MM\Ember%20Media%20Manager\clsTheming.vb",
```

Comment:
In case `xTop.<backcolor>.Value` is null, the `Then` part is not executed.
So line 232 will dereference `tTheme` for the 1st time and I expect it to raise.